build-image-src/Dockerfile-java11

```diff
@@ -65,9 +65,11 @@ ENV JAVA_HOME="/var/lang"
 
 RUN mkdir /usr/local/gradle && curl -L -o gradle.zip https://downloads.gradle-dn.com/distributions/gradle-6.2.2-bin.zip && \
   unzip -d /usr/local/gradle gradle.zip && rm gradle.zip && mkdir /usr/local/maven && \
-  curl -L http://mirror.cc.columbia.edu/pub/software/apache/maven/maven-3/3.6.3/binaries/apache-maven-3.6.3-bin.tar.gz | \
+  curl -L https://mirrors.ocf.berkeley.edu/apache/maven/maven-3/3.6.3/binaries/apache-maven-3.6.3-bin.tar.gz | \
```

Comment:
does downloads.apache.org work? It feels more official than the berkeley mirror

Reply:
Didn't use that as it's called "Backup" on https://www.apache.org/mirrors/ 🤔

Reply:
oh I see, that's fair, thx!